.gitignore

```diff
@@ -1,6 +1,8 @@
 .Rproj.user
 .Rhistory
 .RData
+inst/include/RcppArmadilloConfigGenerated.h
```

Comment:
Because these are generated by their corresponding `.in` files, I think we can just `.gitignore` them.